stack-graphs/src/stitching.rs

```diff
@@ -919,9 +931,25 @@ impl<H: Clone> ForwardPartialPathStitcher<H> {
             }
             self.next_iteration.0.push_back(new_partial_path);
             self.next_iteration.1.push_back(new_cycle_detector);
+            extension_count += 1;
+        }
+
+        let (graph, _, _) = candidates.get_graph_partials_and_db();
+        let end_node = &graph[partial_path.end_node];
+        if end_node.is_root() {
+            self.stats.candidates_per_root_path += candidate_count;
```

Comment:
I think my only style question is whether it's intuitive to use the `+=` operator here.  At first (esp since the rhs is numeric) I thought this was adding `candidate_count` to some total of some kind.  But it's not, it's treating `candidate_count` as an opaque value, and incrementing the count for that value.  Tbh I think a named method will be better here, e.g.

``` rust
self.stats.candidates_per_root_path.record(candidate_count);
```

Reply:
This is a good suggestion, I'll do that!